python/GafferSceneUI/SceneEditor.py

```diff
@@ -134,6 +137,16 @@ def _titleFormat( self ) :
 			_ellipsis = False
 		)
 
+	def __parentChanged( self, widget ) :
+
+		if self.__globalEditTargetLinked or not self.settings().getChild( "editScope" ) :
+			return
+
+		compoundEditor = self.ancestor( GafferUI.CompoundEditor )
+		if compoundEditor and compoundEditor.settings().getChild( "editScope" ) :
```

Comment:
I think the `getChild()` check here is redundant - CompoundEditor will always have an `editScope` plug won't it?

Reply:
Yeah, it will. Removed the check from 9918e2802821385983e69c0e1adf50bb501d2e54